.tool-versions

```diff
@@ -0,0 +1,2 @@
+elixir 1.7.4-otp-21
+erlang 21.0
```

Comment:
```suggestion
elixir 1.10.3-otp-21
erlang 21.0
```

can bump if ya want

won't affect downstream consumers, and will give better compiler mustard